features/flows/itests/src/test/java/org/opennms/netmgt/flows/elastic/MarkerCacheIT.java

```diff
@@ -108,6 +116,9 @@
     @Autowired
     private InterfaceToNodeCache interfaceToNodeCache;
 
+    @Autowired
+    private TransactionTemplate transactionTemplate;
```

Comment:
Use `SessionUtils` instead. It saves you from a lot of boilerplate code

Reply:
:+1:  fixed